### PR TITLE
Update floatfuncs.jl

### PR DESCRIPTION
**Pull request by likanzhan**
Level not correctly rendered (examples were incorrectly rendered as a separate chapter). Is it the right way to correct this?

### CODE REVIEW

File: base/floatfuncs.jl [hunk 1]
```diff
@@ -99,7 +99,7 @@ julia> round(357.913; sigdigits=4, base=2)
     value represented by `1.15` is actually *less* than 1.15, yet will be
     rounded to 1.2.
 
-    # Examples
+# Examples
```
Comment (Keno) [inline base/floatfuncs.jl:102]: I think this example was supposed to apply to the interior of the note, but I'm not sure we have syntax for that. @fredrikekre do you know?

Comment (fredrikekre) [reply base/floatfuncs.jl:102]: Yea, it is correct as is: https://docs.julialang.org/en/v1/base/math/#Base.round-Tuple{Type,Any}

The rendering looks a bit funky but probably a Documenter CSS issue? @mortenpi

Comment (mortenpi) [reply base/floatfuncs.jl:102]: Yeah, I think it's an overlooked edge case (didn't think about headers in admonitions, so it gets the same styling as the main body).

Comment (mortenpi) [reply base/floatfuncs.jl:102]: A workaround would be to turn it into something like a L3/L4 heading or just bold text.

**Pull request closed without merging**